<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalculator</class>
 <widget class="QWidget" name="DepositCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidgetEvents">
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Accrual of interest</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Balance change</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Balance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Event</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
color: black;
border: 2px solid;
border-style: inset;
border-radius: 5px;
border-color: rgba(130, 127, 255, 133);
padding: 0 5px;
background-color: rgba(218, 216, 255, 130);
}

QComboBox QAbstractItemView {
color: black;
border: 0px solid;
border-radius: 0px;
border-color: rgba(130, 127, 255, 133);
background-color: rgb(172, 191, 255);
selection-color: black;
}

QComboBox:item:selected {
background-color: rgb(255, 105, 103)
}

QPushButton {
font: 15pt &quot;Drexs&quot;;
color: black;
border-style: outset;
border-width: 1.5px;
border-radius: 7px;
border-color: rgba(130, 127, 255, 133);
background-color: rgba(218, 216, 255, 130);
selection-background-color: rgba(130, 127, 255, 133);
selection-color: black;
}

QPushButton:pressed {
background-color: rgb(182, 183, 255);
border-style: inset;
}

QLineEdit {
color: black;
border: 2px solid;
border-style: inset;
border-radius: 5px;
border-color: rgba(130, 127, 255, 133);
padding: 0 8px;
background-color: rgba(218, 216, 255, 130);
selection-background-color: rgba(130, 127, 255, 133);
selection-color: black;
}

QLineEdit::input:focus {
/*border: 2px solid #006080;
border-color: rgba(130, 127, 255, 133);*/
background-color: rgba(130, 127, 255, 133);
}

QDoubleSpinBox {
color: black;
padding: 0 3px;
border: 2px solid;
border-style: inset;
border-radius: 5px;
border-color: rgba(130, 127, 255, 133);
background-color: rgba(218, 216, 255, 130);
selection-background-color: rgba(130, 127, 255, 133);
selection-color: black;
}

QSpinBox {
color: black;
padding: 0 3px;
border: 2px solid;
border-style: inset;
border-radius: 5px;
border-color: rgba(130, 127, 255, 133);
background-color: rgba(218, 216, 255, 130);
selection-background-color: rgba(130, 127, 255, 133);
selection-color: black;
}

QLabel {
color: black;
font: 13pt &quot;Drexs&quot;;
}
</string>
     </property>
     <widget class="QLabel" name="labelDeposit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Deposit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTermUnit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>103</width>
        <height>31</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>months</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>years</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelTerm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Term</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStartDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Start date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditStartDate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelRate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interest rate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxCapitalization">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Capitalization</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPeriodicity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pay periodicity</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPeriodicity">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>At end</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Daily</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monthly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quarterly</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Biannually</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Annually</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonCalculate">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>180</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxRate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxDeposit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>1000000000.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
      <property name="value">
       <double>100000.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTerm">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
